base/nvti_tests.c

```diff
@@ -137,6 +153,7 @@ main (int argc, char **argv)
   add_test_with_context (suite, nvti, nvti_new_never_returns_null);
 
   add_test_with_context (suite, nvti, nvti_get_tag_gets_correct_value_one_tag);
+  add_test_with_context (suite, nvti, nvti_set_solution_method_correct);
```

Comment:
This should be on a fresh line below the nvti_get_tag tests.

Reply:
done.